cmd/download.go

```diff
@@ -82,8 +143,9 @@ func init() {
 	downloadCmd.Flags().Bool("generate-imageset", false, "Generate imageset.yaml only")
 	downloadCmd.Flags().Bool("du-profile", false, "Pre-cache telco 5G DU operators")
 	downloadCmd.Flags().Bool("skip-imageset", false, "Skip imageset.yaml generation")
-	downloadCmd.Flags().StringP("hub-version", "", "", "RHACM operator version, in X.Y.Z format")
-	downloadCmd.MarkFlagRequired("hub-version")
+	downloadCmd.Flags().StringP("hub-version", "", "", "(deprecated) RHACM operator version, in X.Y.Z format")
```

Comment:
If --du-profile is set, --acm-version or --hub-version must be set.

Bump cobra to 1.5.0 then we can just use something like:
downloadCmd.MarkFlagsRequiredTogether("du-profile", "acm-version")

Reply:
This would break existing usage with just --hub-version, hence the extra checks during arg validation